scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatOps.scala

```diff
@@ -942,14 +949,18 @@ class FormatOps(
 
   def typeTemplateSplits(
       template: Template,
-      indent: Int
-  ): Seq[Split] = {
+      indent: Int,
+      isDerives: Boolean = false
+  )(implicit line: sourcecode.Line): Seq[Split] = {
     val hasSelfAnnotation = template.self.tokens.nonEmpty
-    val expire = (template.parent match {
```

Comment:
I think this line didn't need to change, sorry didn't notice before

Reply:
Fixed